core/conf/milvus.template

```diff
@@ -158,6 +158,8 @@ logs:
   path: @MILVUS_DB_PATH@/logs
   max_log_file_size: 1024MB
   log_rotate_num: 0
+  log_to_stdout: false
+  log_to_file: true
```

Comment:
This changes should also be added to `config_file_template` at ConfigInit.cpp. Without this, configuration set at runtime may be lost.